build.sbt

```diff
@@ -249,8 +261,8 @@ lazy val bijectionScrooge = {
     libraryDependencies ++= Seq(
       "org.apache.thrift" % "libthrift" % "0.6.1" exclude ("junit", "junit"),
```

Comment:
I wonder if we could move this to util now. @mosesn? Seems like a module there, and/or finagle is the right place.